packages/shared/src/components/profile/SquadMemberList.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { ReactElement } from 'react';
```

Comment:
I think the only difference here with the `UpvoterList` is the property to access `sourceMembers`. We can probably create a ticket to create a generic component to be used by both of the 🚢

Reply:
Yeah agreed, would be a nice win.
Let me add it quickly.

Reply:
Well the one does have a extra button on the right.
But maybe it can be a generic list component?
Added a ticket now.